package.json

```diff
@@ -154,6 +154,7 @@
     "eslint-plugin-react": "^7.34.4",
     "eslint-plugin-react-compiler": "0.0.0-experimental-0998c1e-20240625",
     "eslint-plugin-react-hooks": "^4.6.2",
+    "eslint-plugin-testing-library": "^6.2.2",
```

Comment:
🤔  We should probably make these `dependencies` instead so that X/Toolpad don't have to reinstall them

Reply:
All of the `eslint` ones?

Reply:
Yes, but on second thought, let's keep that for a separate effort. I think for this PR we can keep it. Let's figure out dependencies for the eslint setup separately.

Reply:
Makes sense